setup.py

```diff
@@ -31,6 +31,8 @@
     package_data={"stripe": ["data/ca-certificates.crt"]},
     zip_safe=False,
     install_requires=[
+        'typing_extensions <= 4.2.0; python_version < "3.7"',
+        'typing_extensions >= 4.1.1; python_version >= "3.7"',
```

Comment:
Should these rules be based on 3.6?

Reply:
I don't understand what you mean. We have one rule for python_version 3.7+ and a different rule for < 3.7... IMO the rules should be as unrestrictive as we can make them.

Reply:
I mean that our lowest support version is 3.6. We know that the highest version of  typing_extensions we can rely on is 4.1.1 because only it supports 3.6.

Should the rules be 
`typing_extensions == 4.1.1; python_version == "3.6"`
`typing_extensions >= 4.1.1; python_version > "3.6"` 

?

Reply:
Ah, that's what you have.

Reply:
What you have works.